flink-1.17-connector-bigquery/flink-connector-bigquery/src/main/java/com/google/cloud/flink/bigquery/sink/writer/BaseWriter.java

```diff
@@ -90,6 +91,10 @@ abstract class BaseWriter<IN> implements SinkWriter<IN> {
     // the records in a stream get committed to the table. Hence, records written to BigQuery
     // table is equal to this "totalRecordsWritten" only upon checkpoint completion.
     long totalRecordsWritten;
+    // Counters for metric reporting
+    Counter successfullyAppendedRecordsCounter;
+    Counter successfullyAppendedRecordsSinceChkptCounter;
+    Counter numRecordsInSinceChkptCounter;
```

Comment:
Please remove the `Counter` suffix from these attribute names. It is redundant.

Reply:
Done.